### ISSUE TITLE
FileUploadAdvanced: Cannot add files after clearing with one file

### ISSUE THREAD

**Issue by a839018** [infor-design/enterprise#5598]
**Describe the bug**
I have query regarding FileUploadDdvanced component. Here is the scenario, only one file I need to select at one time, so am selecting one file and if in case am selected the wrong file I need to select second file which is correct. Here is( I have set in setting one file allowed, but after clicking away the error and build. I can not longer upload a new file.

**Steps to reproduce the behavior**
1. Make an example like
```
<div class="row">
  <div class="four columns">
    <div class="fileupload-advanced" data-options="{ maxFilesInProcess: '1', errorMaxFilesInProcess: 'Max (1) files can be process at once'}">
    </div>
  </div>
</div>
```
1. Run it and upload a small file
1. Upload a second file -> correctly shows an error, as per setting one file allowed user will get error cant upload more than one file
1. Then click to remove the error and the file also 
1. Select another file -> this isnt working

**Expected behavior**
1. need some closing error method which will trigger upon closes error
1. needed some pre implemented mechanism to remove existing selected file once we close selected file 

**Version**
- ids-enterprise:v 4.55.0

**Screenshots**

<img width="296" alt="b1" src="https://user-images.githubusercontent.com/90635933/133122103-9d731b52-e12b-4946-90f2-7f5df3920a39.PNG">
<img width="292" alt="b2" src="https://user-images.githubusercontent.com/90635933/133122724-20bb5c80-76e4-43dd-8940-abc977535fb8.PNG">
<img width="291" alt="b3" src="https://user-images.githubusercontent.com/90635933/133122732-cf7557b7-fd6a-4182-b702-0864314d47df.PNG">

**Platform**
 - Infor Application/Infor Hyd M3
 - Device: dell laptop
 - OS Version: Windows 10 
 - Browser Name:  chrome,internet explorer
 - Browser Version: (64-bit)

**Additional context**
need some pre implemented  mechanism in file upload advanced component which will work on file by all aspect.

**Comment by tmcconechy** [comment]
cc @deep7102 for information

1. We should fix the bug so you can upload again after clearing
2. Add additional events for clearing the error and the item and ensure those fire in the new example

**Comment by deep7102** [comment]
Also it need to update in NG, if need anything there.

**Comment by ahivare** [comment]
Hi Team,
Regarding this upload file component issue am checked resolved  "IDS 4.59.4" version but still am facing the same "Cannot add files after clearing with one file" issue. please let me know if am missing anything and help us to resolve this issue.

Thank you.

**Comment by tmcconechy** [comment]
@ahivare 
Try version 4.64.0 which is the latest this is the steps we did to reproduce and fix.

These are the steps and i can verify it works as follows:

- go to https://main-enterprise.demo.design.infor.com/components/fileupload-advanced/example-add-files-after-clearing-a-file.html
- Click Selecta  file and upload a small file (for slow uploading)
- then select another file to trigger the error message
- remove the error message via clicking the close icon (left section)
 -remove also the uploaded file
- then try to upload another file
 -you should be able to upload whenever you remove the old one

So a) try the latest or b) improve the steps to match your case.  Seems to be working fine to what we think is the issue

**Comment by ahivare** [comment]
Hi @tmcconechy 

Okay so issue resolved version is 4.64.0 then may be we have to wait till this version to add in upcoming widget SDK version, as of now we have Widget SDK version 3.12.0 with IDS version 4.59.4.
Thank you for the information.

**Comment by tmcconechy** [comment]
The issue may be resolved in older versions you could try and step back one by one and see. 4.63. 4.62 4.61 ect should be ok too
